Sources/Core/Microsoft.StreamProcessing.Provider/QStreamableVisitor.cs

```diff
@@ -12,11 +13,49 @@ namespace Microsoft.StreamProcessing.Provider
     /// </summary>
     public abstract class QStreamableVisitor : ExpressionVisitor
     {
-        private static readonly MethodInfo AlterDurationMethod = typeof(QStreamableStatic).GetMethod("AlterDuration");
```

Comment:
>AlterDurationMethod [](start = 43, length = 19)

What happened to AlterDuration?

Reply:
It's still there somewhere - was it renamed?

Reply:
Looks completely gone to me

---
In reply to: [309311687](https://github.com/microsoft/Trill/pull/103#discussion_r309311687) [](ancestors = 309311687)

Reply:
It's called SetDuration now. If I recall correctly, I was trying to use "set" when setting things to constant values and "alter" if variable to set a consistent naming scheme.

Reply:
Ah got it, makes sense

---
In reply to: [309313151](https://github.com/microsoft/Trill/pull/103#discussion_r309313151) [](ancestors = 309313151)